src/Expectation.php

```diff
@@ -14,28 +14,28 @@
 /**
  * Class Expectation
  *
- * @method Expectation toBe(mixed $expected) passes if actual === expected
- * @method Expectation toEqual(mixed $expected) passes if actual == expected
- * @method Expectation toBeTruthy() passes if actual is truthy
- * @method Expectation toBeFalsy() passes if actual is falsy
- * @method Expectation toBeEmpty() passes if actual is falsy
- * @method Expectation toBeNull() passes if actual is null
- * @method Expectation toBeA(string $expected) passes if actual is of the expected type
- * @method Expectation toBeAn(string $expected) passes if actual is of the expected type (toBeA alias)
- * @method Expectation toBeAnInstanceOf(string $expected) passes if actual is an instance of expected
- * @method Expectation toHaveLength(int $expected) passes if actual has the expected length
- * @method Expectation toContain(mixed $expected) passes if actual contain the expected value
- * @method Expectation toContainKey(mixed $expected) passes if actual contain the expected key
- * @method Expectation toContainKeys(mixed $expected) passes if actual contain the expected keys (toContainKey alias)
- * @method Expectation toBeCloseTo(float $expected, int $precision) passes if actual is close to expected in some precision
- * @method Expectation toBeGreaterThan(mixed $expected) passes if actual if greater than expected
- * @method Expectation toBeLessThan(mixed $expected) passes if actual is less than expected
- * @method Expectation toThrow(mixed $expected = null) passes if actual throws the expected exception
- * @method Expectation toMatch(string $expected) passes if actual matches the expected regexp
- * @method Expectation toEcho(string $expected) passes if actual echoes the expected string
- * @method Expectation toMatchEcho(string $expected) passes if actual echoes matches the expected string
- * @method Expectation toReceive(string $expected) passes if the expected method as been called on actual
- * @method Expectation toReceiveNext(string $expected) passes if the expected method as been called on actual after some other method
+ * @method \Kahlan\Matcher\ToBe toBe(mixed $expected) passes if actual === expected
```

Comment:
as already in `Kahlan` namespace, it should be doable with `Matcher\ToBe` instead ?

Reply:
Good catch, let's do that :+1: